src/components/dashboard/sidebar/LearnerCreditSummaryCard.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Badge, Row, Col } from '@edx/paragon';
+import dayjs from 'dayjs';
+import {
+  LEARNER_CREDIT_SUMMARY_CARD_TITLE,
+  LEARNER_CREDIT_ACTIVE_BADGE_LABEL,
+  LEARNER_CREDIT_ACTIVE_BADGE_VARIANT,
+  LEARNER_CREDIT_CARD_SUMMARY,
+} from './data/constants';
+import SidebarCard from './SidebarCard';
+
+const LearnerCreditSummaryCard = ({
+  className, subsidyLearnerCredit, searchCoursesCta,
+}) => {
+  const totalRemainingBalanceForUser = subsidyLearnerCredit.reduce(
+    (accumulator, currentSubsidy) => accumulator + (currentSubsidy.remainingBalancePerUser || 0),
+    0,
+  );
+  const subsidyExpiringFirst = subsidyLearnerCredit.sort(
+    (a, b) => new Date(a.subsidyExpirationDate) - new Date(b.subsidyExpirationDate),
+  )[0];
+  return (
+    <SidebarCard
+      title={
+        (
+          <div className="d-flex align-items-center justify-content-between">
+            <h3 className="m-0">{LEARNER_CREDIT_SUMMARY_CARD_TITLE}</h3>
+            <Badge
+              variant={LEARNER_CREDIT_ACTIVE_BADGE_VARIANT}
+              className="ml-2"
+              data-testid="learner-credit-status-badge"
+            >
+              {LEARNER_CREDIT_ACTIVE_BADGE_LABEL}
+            </Badge>
+          </div>
+        )
+      }
+      cardClassNames={className}
+    >
+      {totalRemainingBalanceForUser
+        ? (
+          <p data-testid="learner-credit-text-detailed">
+            Apply your <b>${totalRemainingBalanceForUser}</b>{' '}
+            balance to enroll into courses.
+          </p>
+        )
+        : (
```

Comment:
[clarification] Based on [this Slack thread](https://twou.slack.com/archives/C049C2JGH3L/p1693835644812739), I believe we no longer wish to include the "Apply your $XX balance to enroll into courses." message for either enterprise offers or learner credit policies.